src/beautifiers/Verilog-iStyle-Formatter.coffee

```diff
@@ -0,0 +1,57 @@
+###
+Requires iStyle with istyle-verilog-formatter https://github.com/feilongfl/istyle-verilog-formatter/releases
+###
+
+"use strict"
+Beautifier = require('./beautifier')
+# path = require("path")
+module.exports = class EmacsVerilogMode extends Beautifier
+  name: "Verilog iStyle Formatter"
+  link: "https://github.com/thomasrussellmurphy/istyle-verilog-formatter"
+  isPreInstalled: false
```

Comment:
Remove `isPreInstalled` in favour of `executables`.

This is a bug with https://github.com/Glavin001/atom-beautify/blob/bb9428a4fa483fa6ffe958cdcb519b27e00cb770/src/beautifiers/verilog-mode/index.coffee#L12 too